src/org/usfirst/frc/team4571/robot/subsystems/IntakeSubsystem.java

```diff
@@ -0,0 +1,58 @@
+package org.usfirst.frc.team4571.robot.subsystems;
+
+import org.usfirst.frc.team4571.robot.RobotConstants;
+import com.ctre.CANTalon;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+/**
+ *
+ */
+public class IntakeSubsystem extends Subsystem {
+	private CANTalon intakeMotor;
+	private boolean isRollerIn;
```

Comment:
We dont need to have 2 variables "isRollerIn" and "isRollerOut" because if "isRollerIn" is false it means roller is out. We should get rid of one of the booleans.